test/unit/effects.js

```diff
@@ -17,15 +17,24 @@ QUnit.module( "effects", {
 		this.sandbox = sinon.sandbox.create();
 		this.clock = this.sandbox.useFakeTimers( 505877050 );
 		this._oldInterval = jQuery.fx.interval;
+		this._oldPerformanceNow = window.performance.now;
 		jQuery.fx.step = {};
 		jQuery.fx.interval = 10;
+
+		// Fake performance.now() returning lower values than Date.now()
+		// and that its values are fractional.
+		window.performance.now = function() {
```

Comment:
So extrapolating from your comment https://github.com/jquery/jquery/pull/3151/files/c0d05944f93309dc8c0329316d4b6cf699e21d4d#r66414166 it will not break in IE9, but will break in iOS7 and similar envs